### PR TITLE
Feature/i os push

### PR DESCRIPTION
**Pull request by aderjaan**

### CODE REVIEW

**Comment by meson10** [comment]
@aderjaan Where's the supporting documentation for this? From Parse.com which documents the use of these keys?

**Comment by aderjaan** [comment]
https://parse.com/docs/rest/guide#push-notifications-sending-options

File: core/push.go [hunk 1]
```diff
@@ -14,6 +14,8 @@ import (
 
 const PARSE_URL = "https://api.parse.com/1/push"
 
+const PUSH_SOUND = "default"
```
Comment (meson10) [inline core/push.go]: @aderjaan Is default an accepted value?

Comment (aderjaan) [inline core/push.go]: i'm not sure, dan't find it in the docs, but i guess @ParthDesai found out somewhere. i see some people doing the same http://stackoverflow.com/questions/26488243/no-sound-in-ios-8-parse-push.
Also see https://developer.apple.com/library/ios/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/Chapters/ApplePushService.html

**Pull request merged**